.ci.yaml

```diff
@@ -7,10 +7,7 @@
 #  * https://github.com/flutter/cocoon/blob/main/CI_YAML.md
 enabled_branches:
   - main
-  - master
-  - dev
-  - beta
-  - stable
+  - flutter-\\d+\\.\\d+-candidate\\.\\d+
```

Comment:
How will this get translated to consoles and views?

Reply:
It doesn't. In flutter/infra you have to match beta and stable to a specific version of this.

All this does is saves a step in the release process so you don't need to add the candidate branch each time.